tests/PuphpeteerTest.php

```diff
@@ -127,7 +127,7 @@ public function check_all_resources_are_supported(string $name)
                 }
             }
         } else {
-            var_dump($resource->getResourceIdentity());
+            // dump($resource->getResourceIdentity());
```

Comment:
Was this change made intentionally?

Reply:
I needed it to debug, i can remove it.

Reply:
(before this bunch of commits, there are no var_dump or dump).

Reply:
It would be nice if you clean things like this. And Im ready to merge this.